src/frontend/src/components/LoginContainer/LoginContainer.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import './LoginContainer.css';
+import Header from './Header/Header';
```

Comment:
If there is an index.tsx exporting the component, you don't need to dig deep into `./Header/Header`, `./Header` would work.